doc/source/config.rst

```diff
@@ -52,6 +52,22 @@ Note that command line options `-t`/`-s` can still be used in conjunction with
 `tests` and `skips` given in a config. The result is to concatenate `-t` with
 `tests` and likewise for `-s` and `skips` before working out the tests to run.
 
+Suppressing Individual Lines
+----------------------------
+
+If you have lines in your code triggering vulnerability errors and you are
+certain that this is acceptable in your code they can be individually silenced
```

Comment:
This looks good, just one small nit - having two 'in your code' does not read back so well. I would recommend something like:

"If you have lines in your code triggering vulnerability errors and you are
certain that the code is acceptable, they can be individually silenced
by appending ``# nosec`` to the line::"

Reply:
Good catch. I changed it.